introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-benchmarking/benchmarking/logging.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from typing import Optional
+import sys
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s | %(levelname)s : %(message)s",
+    stream=sys.stdout,
+)
+logging.getLogger("requests").setLevel(logging.CRITICAL)
+logging.getLogger("urllib3").setLevel(logging.CRITICAL)
```

Comment:
i assume this was polluting stdout?

Reply:
Yes, with `connection pool is full, discarding connection` messages. Doesn't impact behaviors.